terraform/eval_diff.go

```diff
@@ -238,6 +238,38 @@ func (n *EvalDiffTainted) Eval(ctx EvalContext) (interface{}, error) {
 	return nil, nil
 }
 
+// EvalFilterDiff is an EvalNode implementation that filters the diff
+// according to some filter.
```

Comment:
Ok gotchya, so it sounds like you're saying its interface will change as we use it in more places.